wagtail-{{ cookiecutter.project_name_kebab }}/.github/publish.yml

```diff
@@ -0,0 +1,36 @@
+# See https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+# for a detailed guide
+name: Publish to PyPI
+
+on:
+  release:
+    types: [released, prereleased]
```

Comment:
Unhelpfully, the docs on the exact nuances of when each of the release triggers is fired is a bit vague https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release so this one is definitely a question: Any thoughts on the choice of `types: [released, prereleased]` as the triggers here? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release seems to suggest

```yaml
on:
  release:
    types: [published]
```

Reply:
Good call!